.github/workflows/verify-pr.yaml

```
@@ -0,0 +1,29 @@
+name: Verify PR
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  verify:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Use Node.js 🐰
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install 💾
+        run: npm install
```

A change from what it was, but might be good to switch to ci for better builds. Although it is just linting...
```suggestion
        run: npm ci
```